src/pages/workspaces/workspace/applications/AppLauncher.js

```diff
@@ -71,6 +72,19 @@ const ApplicationLauncher = _.flow(
         .setStorageLinks(localBaseDirectory, localSafeModeBaseDirectory, cloudStorageDirectory, `.*\\.Rmd`)
     })
 
+    const loadBucketLocation = _.flow(
+      Utils.withBusyState(setBusy),
+      withErrorReporting('Error loading bucket location')
+    )(async () => {
+      const { location } = await Ajax()
+        .Workspaces
+        .workspace(workspace.namespace, workspace.name)
+        .checkBucketLocation(googleProject, bucketName)
+      setLocation(location)
+    })
+
+
+    loadBucketLocation()
```

Comment:
Would it be easier to read if one of the blank lines before `loadBucketLocation()` were moved to after it?

```suggestion

    loadBucketLocation()

```

Reply:
Yes! Fixed.